tools/mkdoc.sh

```diff
@@ -10,4 +10,5 @@ function mkdoc() {
     --mode file
 }
 
-mkdoc docs/typedoc core "Pipcook Interfaces"
+mkdoc docs/typedoc/script core "Pipcook Interfaces"
+mkdoc docs/typedoc/runtime costa "Pipcook Runtime Interfaces"
```

Comment:
```suggestion
mkdoc docs/typedoc/runtime costa "Costa Interfaces"
```

Reply:
Or use the package name?

Reply:
`Core` and `Costa`? It looks more simpler.